lib/src/main/java/com/diffplug/spotless/FeatureClassLoader.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2016 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless;
+
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * This class loader is used to load classes of Spotless features from a search
+ * path of URLs.<br/>
+ * Features shall be independent from build tools. Hence the class loader of the
+ * underlying build tool is e.g. skipped during the the search for classes.<br/>
+ * Only {@link #BUILD_TOOLS_PACKAGES } are explicitly looked up from the class loader of
+ * the build tool and the provided URLs are ignored. This allows the feature to use
+ * distinct functionality of the build tool.
+ */
+public class FeatureClassLoader extends URLClassLoader {
+	static {
+		try {
+			ClassLoader.registerAsParallelCapable();
+		} catch (NoSuchMethodError ignore) {
+			// Not supported on Java 6
```

Comment:
Any value in logging that this functionality is disabled because of the version of Java?

Reply:
Sorry, the comment is misleading. The feature was introduced with Java 7. Will update the comment.